Jenkinsfile

```diff
@@ -46,11 +46,14 @@ pipeline {
                 sh  '''
                     # Clone this repository & Parabank repository into the workspace
                     mkdir parabank-jenkins
-                    git clone https://github.com/whaaker/parabank-jenkins.git parabank-jenkins
+                    git clone -b Pawel-Selenic https://github.com/whaaker/parabank-jenkins.git parabank-jenkins
```

Comment:
When merging back to the main branch, we don't want to continue cloning your Pawel-Selenic branch because it is going to go away after successful merge.  This change should not be part of the pull request nor merged.